tests/unit_tests/test_game.py

```diff
@@ -951,16 +951,76 @@ async def test_game_results(game: Game, players):
     assert len(result_dict["teams"]) == 2
 
     for team in result_dict["teams"]:
-        assert team["outcome"] == (
-            "VICTORY" if team["player_ids"] == [host_id]
-            else "DEFEAT"
-        )
+        outcome, army = ("VICTORY", 0) if team["player_ids"] == [host_id] else ("DEFEAT", 1)
+        assert team["outcome"] == outcome
+        assert team["army_results"] == [
+            ArmyResult(
+                team["player_ids"][0],
+                army,
+                outcome,
+                [],
+            )
+        ]
     assert result_dict["game_id"] == game.id
     assert result_dict["map_id"] == game.map_id
     assert result_dict["featured_mod"] == "faf"
     assert result_dict["sim_mod_ids"] == []
 
 
+async def test_team_game_results(game: Game, game_add_players):
+    game.state = GameState.LOBBY
+    players = (*game_add_players(game, 2, 0), *game_add_players(game, 2, 2))
```

Comment:
Probably doesn't matter, but there is no team with id 0. Team 1 is the free for all team displayed like "-" in lobby, and the rest are all positive integers greater than 1.
```suggestion
    players = (*game_add_players(game, 2, 3), *game_add_players(game, 2, 2))
```